tests/psoc6/hw_ext/uart.py

```diff
@@ -0,0 +1,149 @@
+### UART
+from machine import UART
+import machine
+import binascii
+import time
+import os
+
+# Allocate pin based on board
+machine = os.uname().machine
+if "CY8CPROTO-062-4343W" in machine:
+    uart_rx_pin = "P6_0"
```

Comment:
Okay... it works.
But we need to find other pins as there are used by the SPI already. Can we find another compatible pins? Not used in these diagrams: https://github.com/Infineon/micropython/tree/ports-psoc6-ifx/tests/psoc6/hw_ext/img

Reply:
Yes Updated with other pins.